Frontend/app/utils/AuthHttp.service.ts

```diff
@@ -44,4 +47,21 @@ export class AuthHttp extends Http {
       return Observable.throw(res);
     };
   }
+
+  /**
+  * Service method to save the auth token
+  * @param {String} token
+  */
+  saveAuthToken(token) {
+    // TODO: Should we store the auth token somewhere else?
```

Comment:
Is the auth token that's being stored an autogenerated token? Not a hard coded thing right?